app/src/main/java/fr/free/nrw/commons/profile/leaderboard/LeaderboardFragment.java

```diff
@@ -174,6 +183,19 @@ public void onNothingSelected(AdapterView<?> adapterView) {
         return rootView;
     }
 
+    @Override
+    public void setMenuVisibility(boolean visible) {
+        super.setMenuVisibility(visible);
+
+        // Whenever this fragment is revealed in a menu,
+        // notify Beta users the page data is unavailable
+        if(ConfigUtils.isBetaFlavour() && visible && getContext() != null) {
```

Comment:
Add test when false is passed as the argument, in that case toast should not appear